<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionhopitaux</class>
 <widget class="QDialog" name="gestionhopitaux">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1368</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Hopitaux</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>1211</width>
     <height>631</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Ajouter des hopitaux</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1201</width>
       <height>481</height>
      </rect>
     </property>
     <property name="title">
      <string>Ajout</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>ID Hopital</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Capacite</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>140</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Nom Hopital</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="id_hopital">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>50</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_hopital">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>140</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="adresse">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>240</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLineEdit" name="capacite">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>350</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="boutton_ajouter">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>150</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>820</x>
        <y>270</y>
        <width>221</width>
        <height>171</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/new/prefix1/logo.png);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>420</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Myanmar Text</family>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Image</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>430</y>
        <width>351</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>410</y>
        <width>231</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>427</y>
        <width>201</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Selectionne une image</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Afficher les hopitaux</string>
    </attribute>
    <widget class="QTableView" name="tab_hopitaux">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>771</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="search">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="search_button">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>20</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Search</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_pdf">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>150</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>PDF</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>930</x>
       <y>251</y>
       <width>161</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <item>
      <property name="text">
       <string>ID_HOPITAUX</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>NOM</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>ADRESSE</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>790</x>
       <y>255</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>      Trier par :</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_5" native="true">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>370</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/logo.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Supprimer des hopitaux</string>
    </attribute>
    <widget class="QPushButton" name="boutton_supprimer">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>180</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>200</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID_Hopital</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_hopital_2">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_hopitaux_3">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>10</y>
       <width>521</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
    <widget class="QWidget" name="widget_6" native="true">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>350</y>
       <width>191</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/logo.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_4">
    <attribute name="title">
     <string>Modifier des Hopitaux</string>
    </attribute>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ID Hopital</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nom Hopital</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>210</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Adresse</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>290</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>11</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Capacite</string>
     </property>
    </widget>
    <widget class="QPushButton" name="boutton_modifier">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>160</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="id_hopital_3">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>50</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="nom_hopital_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>130</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="adresse_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>210</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="capacite_2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>290</y>
       <width>261</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>300</y>
       <width>221</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/logo.png);</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_hopitaux_2">
     <property name="geometry">
      <rect>
       <x>680</x>
       <y>10</y>
       <width>511</width>
       <height>471</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>Statistique des hopitaux</string>
    </attribute>
    <widget class="QFrame" name="horizontalFrame">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>1191</width>
       <height>491</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>Chat</string>
    </attribute>
    <widget class="QTextEdit" name="textEdit">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>1161</width>
       <height>341</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="nickname">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>11</y>
       <width>441</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bind">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>470</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Connecter</string>
     </property>
    </widget>
    <widget class="QPushButton" name="send">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>410</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Envoyer</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="message">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>420</y>
       <width>561</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Nickname:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>450</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Message:</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>1040</x>
       <y>400</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/new/prefix1/logo.png);</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_8">
    <attribute name="title">
     <string>Arduino</string>
    </attribute>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Lampe:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>ON</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>40</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>280</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Etat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Myanmar Text</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ETAT:</string>
     </property>
    </widget>
    <widget class="QTableView" name="tab_hopitaux_4">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>130</y>
       <width>771</width>
       <height>411</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
